javascript/sdk/lib/controllers/partner.ts

```diff
@@ -112,4 +112,20 @@ export default class PartnerController {
 
     return (await response.json()) as DeployedEndpoints;
   }
+
+  /** Generate webhook authenication information for specified partner */
+  async webhookAuth(
+    partnerCode: ControlCode,
+    options: RequestOptions = {}
+  ): Promise<StatusResponse> {
+    const response = await this.#client.requestWithAuth(
+      `/partner/suppress/${partnerCode}`,
+      {
+        method: "GET",
+        ...options,
+      }
+    );
+
+    return response.json();
```

Comment:
this might need to be awaited?

```suggestion
    return await response.json();
```

Reply:
Make edits as necessary

Reply:
updated btw